packages/syntax-core/src/Avatar/Avatar.tsx

```diff
@@ -1,14 +1,66 @@
+"use client";
```

Comment:
@somethiiing Could we remove this for now? This will break once we try to use it in Cambly-Frontend https://github.com/Cambly/syntax/pull/274